pkg/k8s/client/client.go

```diff
@@ -46,16 +47,8 @@ var (
 func GetLocal(k8sContext string) (*kubernetes.Clientset, *rest.Config, string, error) {
 	if client == nil {
 		var err error
-
-		clientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
-			clientcmd.NewDefaultClientConfigLoadingRules(),
-			&clientcmd.ConfigOverrides{
-				CurrentContext: k8sContext,
-				ClusterInfo:    clientcmdapi.Cluster{Server: ""},
-			},
-		)
-
-		namespace, _, err = clientConfig.Namespace()
+		clientConfig = GetClientConfig(k8sContext)
+		namespace, err = GetNamespace(k8sContext)
```

Comment:
Since we have already initialized the namespace, we can probably remove `namespace` from this function

Reply:
I think this change may break other commands, `GetLocal` method is used by all the other commands, each command only needs some of the variables returned by `GetLocal`. It would be convenient to make a refactor of `GetLocal` method so that each command collects only the variables it requires and does not compute the others.

Reply:
Sounds good, we can do it in a different PR

Reply:
I think this change may break other commands, the `GetLocal` method is used by all other commands, each command only needs some of the variables returned by `GetLocal`. It would be convenient to make a refactor of the GetLocal method so that each command collects only the variables it requires and does not compute the others.